cmd/karmada-search/app/options/options.go

```diff
@@ -122,3 +131,14 @@ func (o *Options) Config() (*search.Config, error) {
 	}
 	return config, nil
 }
+
+func serveHealthzAndMetrics(address string) {
```

Comment:
Karmada-search is implemented on genericapiserver. It has implemented the healthz probe, default port is 443. You can based on `vendor/k8s.io/apiserver/pkg/server/config.go`.

Reply:
OK